test/templates/app-ts.test.ts

```diff
@@ -0,0 +1,55 @@
+import { fastify } from 'fastify'
+import { test } from 'tap'
+const sgetOriginal = require('simple-get').concat
+
+import appDefault, { app } from '../../templates/app-ts/src/app'
+
+const PORT = 3001;
```

Comment:
```suggestion
const PORT = 0;
```

Reply:
use a random a picked port

Reply:
Done. Needed slight adjustment in test itself as well.